nss-day-cohort-39/tabloid-full-stack-vanjies
# Tl delete category

Completed category delete function. I should've saved my sql queries to get it working, but it's just one column add.
To Test:
1. Add Active column in Category table, make it BIT, NOT NULL, and set the Default to 1
ALTER TABLE Category ADD Active BIT NOT NULL DEFAULT 1;
I believe the above should do it. **Make sure the column is named "Active", not "isActive"**
2. Once the database is refreshed, route to the Category Management tab.
3. Ensure the delete function works. This will keep the Category in the database, but will make it inactive.
4. Add a new category and ensure it can be deleted as well.
5. Start deleting a category but say No to route back to the Category Management page.
6. Route to New Post and ensure only categories that are active can be chosen.
Note: The posts that are assigned categories that have been deleted will still have those categories attached to them.
